build.gradle

```
@@ -13,15 +13,15 @@ configurations {
     provided
 }
 
-version = "0.1.16"
+version = "0.1.17"
```

IDEA: `${pluginVersion}.sdk.{sdkVersion}`

```suggestion
version = "0.1.17.sdk.16.0.2"
```

---

0.1.17で大丈夫です！